tools/VagrantSetup/Vagrantfile_nightly

```diff
@@ -14,7 +14,7 @@ Vagrant.configure(2) do |config|
     inline: "docker stop lanraragi || true && docker rm lanraragi || true"
   config.vm.provision "docker" do |d|
     d.pull_images "difegue/lanraragi:nightly"
```

Comment:
```suggestion
    d.pull_images "difegue/lanraragi"
```

Reply:
Didn't know that the :nightly suffix wasn't effective here -- It's probably better to remove it altogether here so we don't have double :nightly in the following run command.